docs/conf.py

```diff
@@ -178,6 +178,13 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'togadoc'
 
+try:
```

Comment:
This works with read the docs but allows building and testing the documentation locally by installing the sphinx-rtd-theme package which provides the readthedocs theme so the locally built documentation looks the same as on readthedocs.io when built locally.

The try/except prevents a doc build failure if the sphinx-rtd-theme package is not installed.  To prevent causing unexpected breakage due to the behavior change.